backend/controller/observability/ingress.go

```diff
@@ -0,0 +1,76 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/observability"
+)
+
+const (
+	ingressMeterName         = "ftl.ingress"
+	ingressMethodAttr        = "ftl.ingress.method"
+	ingressPathAttr          = "ftl.ingress.path"
```

Comment:
I'm curious if this is too high of a cardinality to log

Reply:
Depending on what info is contained in the paths, we could potentially chop the path and keep just the top level prefix, or otherwise parse out only the data that we find useful

Reply:
Oh huh if the path is just the module name, I guess it doesn't even matter if we do anything special here, because it won't add any cardinality beyond the verb ref. We can just leave this as is - it's probably easiest to understand this way

Reply:
The path is entirely user defined, i.e. `//ftl:ingress GET /http/echo` -> `https://<domain:port>/http/echo`

But you're right it's equivalent to adding a verb ref, and it's pretty usefull; I'll leave it in.

Reply:
Ahh got it, that makes sense. Sounds good!